src/sections/FAQs.tsx

```diff
@@ -1,5 +1,101 @@
+import { useState } from 'react';
+
+const Faq = () => {
+  const [openIndex, setOpenIndex] = useState<number | null>(null);
+
+  // Function to toggle FAQ open/close
+  const toggleFaq = (index: number) => {
+    setOpenIndex(openIndex === index ? null : index);
+  };
+
+  // FAQ content
+  const faqs = [
+    {
+      question: 'How do you selectively reveal content in an email?',
+      answer:
+        "We can hide any information or selectively reveal any text, wether that’s the sender, receiver, subject, body etc using Regex. Regex is short for regular expression, this term represents sequence of characters that forms a search pattern, commonly used for string matching within text. It consists of a sequence of characters that define a search pattern, enabling complex searches, substitutions, and string manipulations. For example, in programming and text processing, regex can identify, extract, or replace specific text patterns, such as email addresses, dates, or phone numbers, by defining these patterns through a combination of literal characters and special symbols. In the context of ZK Email it is used to parse email headers and extract relevant information."
+    },
+    {
+      question: 'How can I do it anonymously?',
+      answer: 'ZK Email leverages the principles of Zero Knowledge proofs and serverless execution within the browser to provide a verifiable yet anonymous way to confirm an emails contents and recipents . Zero Knowledge proofs allow provers to prove that they know or possess certain information without revealing the information itself to a verifer. In ZK Email, this technology is used to verify user identity and email content without exposing sensitive data to a server or other users. The serverless architecture means that all proof generation is executed entirely within the user`s browser.'
+    },
+    {
+      question: 'Why don’t I need to trust you?',
+      answer: 'The DKIM email signitures do not contain information we can use to sign other emails, all our code is open source and being audited'
+    },
+    {
+      question: 'How do you verify the email contents?',
+      answer: 'We use the existing DKIM Signiture, almost all emails are signed by the sending domain server using an algorithm called DKIM. It can be summarized as this rsa_sign(sha256(from:..., to:..., subject:..., <body hash>,...), private key). Every time an email is sent we can verify the sender, receiver, the subject, the body by checking if the corresponding public key of the email address applied on the DKIM signiture returns the sha256(from:..., to:..., subject:..., <body hash>,...) a hash of the from, to subject, email details which we can check by rehashing'
+    }
+  ];
+
+  return (
+    <div className="w-full mx-auto mt-10 z-10 px-8 sm:px-36">
+      <div className='flex flex-row justify-between'>
+        <div>
+          <h1 className="text-3xl font-semibold mb-6 h3 text-left " style={{textAlign:'left'}}>FAQs</h1>
```

Comment:
Why are you repeating textAlign property in both tailwind and CSS in JSX? Remove it from one place else it can cause confusions while trying to change this property in the future